test/new_relic/load_test.rb

```diff
@@ -4,7 +4,7 @@
 
 require_relative '../test_helper'
 require 'resolv'
-require 'mocha/setup'
+require 'mocha/minitest'
```

Comment:
`require 'mocha'` and `require 'mocha/setup'` are deprecated. Users are encouraged to perform a more specific `require`, such as `mocha/minitest` in our case.